Src/BootCamp.Chapter/CVSWriter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class CVSWriter
+    {
+        public static void WriteCityData(string file, string headertext, string outcome)
+        {
+            File.WriteAllText(file, outcome); 
+
+        }
+
+        public static void WriteTimeData(string file, string headertext, List<OutputTime> outcome)
+        {
+            var encoding = new UTF8Encoding();
+            File.WriteAllText(file,$"Hour, Count, Earned{Environment.NewLine}");
+            CultureInfo dutch = new CultureInfo("nl-NL");
+            dutch = (CultureInfo)dutch.Clone();
```

Comment:
Not needed.

Reply:
not needed. I use this to make it work that the currency symbol is placed behind the amount.
Without it , it will be € 5 instead of  5 €